README.md

```diff
@@ -1,43 +1,62 @@
 # Executable Tutorials
 
-A repository demonstrating one way to manage and distribute interactive tutorials.
-See the [Guide to Contributing](https://scentific-python.org/executable-tutorials/contributing.html)
-for a good overview of what this is like to use.
+## Use Cases
 
-## Demo Links
+Many organizations maintain collections of tutorials addressing realistic
+problems in a science domain, with runnable code examples written and
+kept current by experts in the tools. They are useful as:
 
-- **[Tutorials](https://github.com/scientific-python/executable-tutorials/tree/main/tutorials)** are written in MyST Markdown.
-- Tutorials can be easily edited as notebooks in JupyterLab.
-- Executed tutorials are **[published](https://scientific-python.github.io/exeuctable-tutorials/)** on a GitHub Pages site.
-- **[Jupyter Lite (beta)](https://scientific-python.github.io/executable-tutorials/jupyterlite/lab/index.html)** (works only on the basic executable example so far, missing dependencies for others)
-- **[Binder](https://mybinder.org/v2/gh/scientific-python/executable-tutorials/notebooks)**
+- A trove of working snippets to copy and paste
+- A maintained resource for self-guided learning
+- A ready-to-use curriculum for interactive workshops
+- A suite of science domain-specific "integration tests"
 
-### Notes on the notebook file format
+## Key Features
 
-We do use MyST Markdown format for the notebooks, please visit [the upstream documentation](https://mystmd.org/guide/md-vs-ipynb) to learn more about the reasoning behind the choice.
-However, we add it here that you can easily have the same user experience in JupyterLab if these two dependencies are installed:
- - [jupytext library](https://pypi.org/project/jupytext/)
- - [jupyterlab-myst JupyterLab extension](https://pypi.org/project/jupyterlab-myst/)
+This repository demonstrates one way to configure a collection of tutorials,
+covering some core features:
 
+- Source is in [MyST Markdown][] which is easy for humans to edit and review.
+- The executed examples---code and results---are published as a static site.
+  ([example][static site example])
```

Comment:
Having a bit of "click here" vibe from the example links, maybe work them into the text here and below?

https://heyoka.medium.com/dont-use-click-here-f32f445d1021